examples/datasource-http/src/DataSource.ts

```diff
@@ -41,11 +41,20 @@ export class DataSource extends DataSourceApi<MyQuery, MyDataSourceOptions> {
        * }
        */
       const datapoints = response.data.datapoints;
+      if (typeof datapoints === 'undefined') {
+        throw new Error('Remote endpoint reponse does not contain "datapoints" property.');
+      }
 
       const timestamps: number[] = [];
       const values: number[] = [];
 
       for (let i = 0; i < datapoints.length; i++) {
+        if (typeof datapoints[i].Time === 'undefined') {
```

Comment:
In this specific case you could argue, that `Time` might not be initialized but you are actually accessing a property of an already initialized variable `datapoints[i]` that might not exist and thus return undefined.